Cargo.toml

```diff
@@ -14,21 +14,23 @@ exclude = ["/.github", "/examples", "/scripts"]
 
 [dependencies]
 tokio = "1.0"
-rustls = { version = "=0.22.0-alpha.2", default-features = false }
+rustls = { version = "0.22.0-alpha.2", default-features = false }
```

Comment:
We want to stick to using `=` for semver pre-releases, because Cargo otherwise treats pre-releases as compatible even though they often aren't. If you want to pull in the latest rustls changes, please submit a version bump PR in the rustls repo to request another alpha.